alcs-frontend/src/app/features/application/application-meeting/application-meeting-dialog/application-meeting-dialog.component.scss

```diff
@@ -6,3 +6,24 @@
 .flex-child {
   flex: 1 0 auto;
 }
+
+mat-radio-button {
+  display: block;
+  margin: 4px 0;
+}
+
+.reason-wrapper {
+  margin: 16px 0;
+}
+
+.radio-group-label {
+  display: block;
+  margin-bottom: 4px;
+}
+
+label {
+  font-weight: 500;
```

Comment:
Kinda sad to see another custom font, seems the only place we have used 500 so far.

Reply:
I'll create a ticket to address typography across the app. All this fonts are already used somewhere in app, just copied them here for consistency